ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py

```diff
@@ -2525,6 +2525,31 @@ def test_jax_numpy_around(
     )
 
 
+# round
+@handle_frontend_test(
+    fn_tree="jax.numpy.round",
+    dtype_and_x=helpers.dtype_and_values(available_dtypes=helpers.get_dtypes("float")),
+    test_with_out=st.just(False),
```

Comment:
since this function accepts `out` argument, you should remove this

Reply:
fixed